### ISSUE TITLE
Add sclerosis of iliac bone

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10527]
**Preferred term label:**
Sclerosis of iliac bone

**Synonyms**
Sclerotic ilium, Increased bone density of the ilium

**Definition (free text, please give PubMed ID)**
An elevation in bone density in the ilium. Sclerosis is normally detected on a radiograph as an area of increased opacity. PMID: 22084176

**Parent term (use hpo.jax.org/app)**
Increased bone mineral density HP:0011001
Abnormal ilium morphology HP:0002867

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
McCune-Albright syndrom (omim174800), PMC6611735

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000869
- label: Sclerotic ilium
- parents: [Abnormal ilium morphology (HP:0002867)](https://hpo.jax.org/app/browse/term/HP:0002867)
- pmid: PMID:22084176
- def: An elevation in bone density in the ilium. Sclerosis is normally detected on a radiograph as an area of increased opacity.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10527
- synonyms: Increased bone density of the ilium(NONE)|Sclerosis of iliac bone(NONE)
